packages/circuits/src/regexes/invitation_code.json

```diff
@@ -2,7 +2,7 @@
     "parts": [
         {
             "is_public": false,
-            "regex_def": "CODE:0x"
+            "regex_def": "ACCOUNTKEY.0x"
```

Comment:
i think this should still be code?

Reply:
We thought of using `ACCOUNTKEY` in #43 to be more explicit - and CODE might also occur in a non-email-wallet email (extra precaution now that we don't have user<->relayer registration before a realyer can execute EmailOp).

Is `CODE` better in terms of readability? We can also have space as additional delimiter for subject? `Create Email Wallet with AccountKey 0x123` 🤔